amr.md

```diff
@@ -2768,22 +2765,29 @@ Annotators are encouraged to use `have-degree-91` as the root concept (as oppose
 
 Additionally, `have-degree-91` is used for the degree-consequence construction, which licenses an argument representing the result or consequence of the degree to which a state is true: 
 
->It is too early to reach any conclusion as to the motive and identity of the attackers.
+
 ```lisp
-(h / have-degree-91
-      :ARG2 (e / early)
-      :ARG3 (t / too)
-      :ARG6 (c2 / conclude-01 :polarity -
-            :ARG1 (a / and
-                  :op1 (m / motive
-                        :poss (p / person
-                              :ARG0-of (a2 / attack-01)))
-                  :op2 (i / identity
-                        :poss p))))
+(h / have-degree-91 
+      :ARG2 (e / early) 
+      :ARG3 (t / too) 
+      :ARG6 (c / conclude-01)) 
 ```
 
-Note: The primary relation of the Consequence argument may be elided or unspecified, as it is above.  Annotators should introduce the most logical relation fitting with the context, including the possible introduction of negative polarity, which often accompanies Degree-Consequence constructions with “too” (above: it’s too early; therefore, there’s no conclusion…).  See AMR Dictionary for additional examples. 
+>It is too early to reach any conclusion.
+
+Note: The primary relation of the Consequence argument may be elided or unspecified, as it is here.  Annotators should introduce the most logical relation fitting with the context.  Although it may be tempting to introduce modality, possibility, and/or polarity in the consequence (i.e. expressing that the watch does NOT fit), a survey of these constructions has demonstrated the difficulty in introducing such elements consistently in context.  Therefore, annotators are asked to limit the Arg6 to the simple relation to which the degree has reference.  See AMR Dictionary for additional examples. Negative polarity should only be introduced (modifying have-degree-91) when it is explicit: 
```

Comment:
"expressing that the watch does NOT fit": what watch?

Reply:
sorry, copy paste error -- will fix.